quickstart.sh

```diff
@@ -1,17 +1,39 @@
+# Prerequsites
+# 1. GCP billing account name found at https://console.cloud.google.com/billing, in the examples this is `My Billing Account FY22`
+# 2. Desired project ID name, in the examples this is `cvah-helk-training-fy22`
+
+# Install deps
+# curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+# sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+# sudo apt-get update && sudo apt-get install terraform packer
+
 # Create a new project
 pushd ./terraform/setup
-terraform init 
+# Do a plan to ensure configs are set up
+# terraform init -var "gcp_billing_account_name=My Billing Account FY22" -var "project_id=cvah-helk-training-fy22"
+# terraform plan -var "gcp_billing_account_name=My Billing Account FY22" -var "project_id=cvah-helk-training-fy22"
+# terraform apply -auto-approve -var "gcp_billing_account_name=My Billing Account FY22" -var "project_id=cvah-helk-training-fy22"
+terraform init -auto-approve
 terraform apply -auto-approve
 popd 
 
+# If in GCP Console, set the project now that it's been created
+# gcloud config set project cvah-helk-training-fy22
+
 # Create the custom image 
 pushd ./packer
-packer init 
+# Specify the same Project ID as above
+# packer init -var "project_id=cvah-helk-training-fy22" build.pkr.hcl
```

Comment:
`packer init` does not require the variables to be passed however, it doesn't seem to break anything.